packages/react-ux-capture-example/src/DurationLabel.jsx

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
```

Comment:
Not a blocker but we should definitely have a ticket to add unit tests. This is a larger demo app than I anticipated! Which is great, but also means it's likely to break.

For components like this, even if the unit tests were just a snapshot, I think that'd be great.

Reply:
Agreed, was hitting it trying to do refactoring, will try to add those.

Reply:
Created https://meetup.atlassian.net/browse/WP-995